less/home-page.less

```diff
@@ -1,39 +1,47 @@
 // Home Content
-
-.home {
-
   .intro {
 
     h2 {
       padding: 15px;
     }
-
     p {
       padding:0 15px;
     }
-  
+    .header-image {
+      border-radius: 4px;
+    }
   }
 
   .space-images {
     display: flex;
     flex-wrap: wrap;
     width: 100%;
-
+    .blast {
+      @media @phone {
+        width: 92%;
+        margin: 5% 5%;
+      }
+    }
+    .sky, .launch {
+      .mixin-set();
```

Comment:
nice way to get your images to disappear in mobile by using the mixin.   It may be easier to follow to rename the mixin to disappear() or something along those lines.